pkg/controllers/cassandra/cassandra.go

```diff
@@ -172,6 +182,22 @@ func init() {
 					},
 				),
 			),
+			ctx.SharedInformerFactory.InformerFor(
+				ctx.Namespace,
+				metav1.GroupVersionKind{
+					Group:   navigator.GroupName,
+					Version: "corev1",
+					Kind:    "Service",
```

Comment:
`corev1` is not a version, and services do not exist in the navigator group.

`Group: ""`
`Version: "v1"`

(see similar examples in the ElasticsearchCluster controller)
(as discussed, I'm aware that this way of using the SharedInformerFactory sucks, and I promise it won't hang around forever, but whilst it does we need to make sure to use it correctly!)

Reply:
Deon.